lib/client.js

```diff
@@ -467,6 +480,11 @@ MqttClient.prototype.subscribe = function () {
     messageId: this._nextId()
   }
 
+  if (!subs.length) {
+    callback(null, [])
```

Comment:
@mcollina Will this break the API? I return an empty array if all subscriptions are duplicates